### PR TITLE
R3C2: Rules & Regs v1.2

### PR DESCRIPTION
**Pull request by sdcaulley**
First pass at updating content.

### CODE REVIEW

File: docs/rules-and-regulations.md [hunk 1]
```diff
@@ -17,204 +16,101 @@ The Organization shall not arbitrarily discriminate on the basis of race, nation
...
+3. Community
+4. Network
  
 ### Section 2.02. Membership Qualifications.  
 
-Any person may become and remain a member of this Organization by:
-1. Being sponsored by an existing member; and
-2. Meeting the qualifications set forth for each Membership Classification; and  
-3. Making full payment of any non-refundable membership fee as set forth in Section 2.06 of these Rules and Regulations. 
+Any person may become and remain a member of this Organization.
```
Comment (edjieun) [inline docs/rules-and-regulations.md:25]: I think this needs to be separated into parts. TBD.  Good until the next iteration.

To consider: 
1.  Any person may become a member of this Organization (Collective)
2.  Defining Membership in good standing (active)(inactive), criteria to be in bad standing needs to be defined in a code of conduct section.
3.  What is the code of conduct, and what are the consequences.

File: docs/rules-and-regulations.md [hunk 2]
```diff
@@ -17,204 +16,101 @@ The Organization shall not arbitrarily discriminate on the basis of race, nation
...
+### Section 2.08. Voluntary Withdrawal.
  
+All members shall have the right to resign from the Organization and terminate his or her membership by filing with the Organization written notice of resignation. The resignation shall become effective immediately without any action on the part of the Organization. Upon resignation, such member shall be entitled to receive all compensation and other monies earned up to the date of the written notice of resignation.
```
Comment (edjieun) [inline docs/rules-and-regulations.md:58]: Leaving a comment for now, will attempt an edit next.  I'm still familiarizing myself with GitHub.

We should change "his or hers" to theirs.  We are referring to members in the plural, and we do not need to add gender.

File: docs/rules-and-regulations.md [hunk 3]
```diff
@@ -17,204 +16,101 @@ The Organization shall not arbitrarily discriminate on the basis of race, nation
...
-
-Any Business System which may be procured by the Organization free of charge which is requested by at least two members may be procured at the discretion of the Business Systems Point in compliance with section 7.02 of these Rules and Regulations. 
-
-### Section 7.06. Paid for Business Systems.
-
-Once a Business System has been approved by a majority of the Partner Members, or is recommended by the Business System Point, it must also be approved by the OP, Business Systems Point, and Finance Point prior to the purchase of such Business Systems. 
-
-Access to Paid Systems may be rationed when needed (in order to manage costs) according to an objective calculation approved by the consent of 75% of the OP, Business Systems Point, and Finance Point.
-
+The approval of Business Systems is done by the Foundation team and the currenly Partners. If it is a system requiring payment the approval proccess will include the Finance team.
```
Comment (edjieun) [inline docs/rules-and-regulations.md:114]: Will make minor edits to correct typo.

**Review by edjieun — COMMENTED** [review]
@sdcaulley I made some comments.  I want to make a couple of small edits on my next pass.

File: docs/rules-and-regulations.md (hunk 3 above)
Comment (sdcaulley) [reply docs/rules-and-regulations.md:114]: Did you make the edit?  If not, what did you find and I can change it.

**Comment by hankish** [comment]
I just pushed an update. Tweaked the content a bit. I just removed the business systems part... it's super outdated. And most of the rest is now covered elsewhere in the repo (though `structure.md` is not yet in this branch but I linked it anyway). 

It's kind of sparse now! Which I think is actually a good sign that the gov repo is coming along and is well segmented in re-referencable files. But we should probably add some more values and standards pieces back in. Honesty, ethical behavior, explaining conflicts of interest, etc.

We have some of this language in other docs somewhere already... it's just not in here yet.

**Review by traklord — APPROVED** [review]
I approve!

**Comment by hankish** [comment]
Here is the video from the review & approval of this:
- [2024-08-28 - Session 2 - Contribution Review of Multiple Release 3 Branches](https://drive.google.com/file/d/11MWHNR5U9oxnL4pT4is_5kxokk3DTNMQ/view)

**Pull request merged**